single-page.bs

```diff
@@ -376,10 +376,19 @@ urlPrefix: https://www.w3.org/TR/CSP3/#; type: dfn; spec: CSP
 
 <!-- ********************************** CSSOM-VIEW ********************************************* -->
 
-urlPrefix: http://www.w3.org/TR/cssom-view/#; type: interface; spec: CSSOM-VIEW;
-    text: screen;
+urlPrefix: http://www.w3.org/TR/cssom-view/#; spec: CSSOM-VIEW;
+    text: screen; type: interface
+    urlPrefix: eventdef-document-; type: event;
+        text: scroll
+        text: resize
+    text: run the resize steps; type: dfn
```

Comment:
You might want to pull up the type: dfn a level and nest all these text: items under that dfn section

type: dfn;
    text: run the resize steps
    text: run the scroll steps
etc...